README.md

```diff
@@ -53,13 +53,13 @@ Versions of supported components
 --------------------------------
 
 
-[kubernetes](https://github.com/kubernetes/kubernetes/releases) v1.6.7 <br>
-[etcd](https://github.com/coreos/etcd/releases) v3.0.17 <br>
+[kubernetes](https://github.com/kubernetes/kubernetes/releases) v1.7.3 <br>
+[etcd](https://github.com/coreos/etcd/releases) v3.2.4 <br>
 [flanneld](https://github.com/coreos/flannel/releases) v0.8.0 <br>
-[calicoctl](https://github.com/projectcalico/calico-docker/releases) v0.23.0 <br>
+[calicoctl](https://github.com/projectcalico/calico-docker/releases) v1.1.3 <br>
```

Comment:
Also, `roles/uploads/defaults/main.yml` and `roles/download/defaults/main.yml` seams to differ with respect to Calico version.